Week1_Knowledge.md

```diff
@@ -77,6 +77,8 @@ Conclusion: If _P_ is false, then _Q_ can be anything.
 Equivalent to:
 _P_ → _Q_ ∧ _Q_ → _P_
 
+Only true if P and Q are the same.
```

Comment:
Do you want to change the wording of this because right now it seems like you're saying the actual expression: _P_ → _Q_ ∧ _Q_ → _P_ is only true if P and Q are the same. What you're really trying to say is that biconditionals evaluate to true when P and Q are the same.